packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -347,36 +345,29 @@ const RelationInput = ({
 };
 
 const ReactQueryRelationResult = PropTypes.shape({
```

Comment:
I think we shouldn't call the type any longer `ReactQueryRelationResult`, because it doesn't contain a full react-query response. What about

```suggestion
const RelationsResult = PropTypes.shape({
```

The reason why I wanted to pass in a paginated result is, that loadMore should handle scrolling at some point (users clicks "load more" -> scroll to the end of the fetched page), but I guess we could start to pass the pagination information on a different way.

Reply:
Is this a ticket? This is something we can look at during `relations-reordering` and adjust accordingly, I can't imagine it would be crazy hard.